public/app/config/errors.js

```diff
@@ -6,11 +6,13 @@ const ERROR_JSON_CORRUPTED = 'ERROR_JSON_CORRUPTED';
 const ERROR_SPACE_ALREADY_AVAILABLE = 'ERROR_SPACE_ALREADY_AVAILABLE';
 const ERROR_DOWNLOADING_FILE = 'ERROR_DOWNLOADING_FILE';
 const ERROR_GENERAL = 'ERROR_GENERAL';
+const ERROR_DUPLICATE_CLASSROOM_NAME = 'ERROR_DUPLICATE_CLASSROOM_NAME';
```

Comment:
Technically it is possible to create two users with the same username, but since the teacher will only see the usernames he will never be able to see the difference between the two users (eg. "do you wish to assign this data to X or X?")

Reply:
I see the point about the user. However, couldn't we have two classrooms with the same name? I thought that this is what this error was mentioning.